user-service/src/main/java/uk/gov/gchq/palisade/service/user/response/ErrorResponse.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.user.response;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Error message in human readable form.  This can be generated in any of the services.  Once an error occurs
+ * in a service,  processing of the requests stops.  This messaging is constructed and forwarded to the Results
+ * Service skipping any services that have not been preformed.  Results services will forward this message back
+ * to client who should be given enough information to correct the problem before tying again.
+ */
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class ErrorResponse {
+
+    //This version of the message may not necessarily be sanitised.  It will need to pass information to
+    // the Audit Service that may technical details as to the reason for the failure.  This can then be used to
+    // resolve the issue if it was technical in natured.
```

Comment:
```suggestion
    // the Audit Service that may include technical details as to the reason for the failure.  This can then be used to
    // resolve the issue if it was technical in nature.
```